redi-connect-front/src/pages/app/profile/mentor/ProfileMentor.tsx

```diff
@@ -1,100 +1,104 @@
-import { createStyles, Grid, Theme, withStyles } from '@material-ui/core'
 import React from 'react'
-// import { Avatar } from '../../../../components/Avatar'
-import { CategoryChip } from '../../../../components/CategoryChip'
-import { ProfileAvailableMenteeSlots } from '../../../../components/ProfileAvailableMenteeSlots'
-import { ProfileLanguages } from '../../../../components/ProfileLanguages'
-import { ProfileName } from '../../../../components/ProfileName'
-import { ProfileOccupation } from '../../../../components/ProfileOccupation'
-import { ProfileWorkPlace } from '../../../../components/ProfileWorkPlace'
+import Avatar from '../../../../components/organisms/Avatar'
+// import { CategoryChip } from '../../../../components/CategoryChip'
+// import { ProfileAvailableMenteeSlots } from '../../../../components/ProfileAvailableMenteeSlots'
+// import { ProfileLanguages } from '../../../../components/ProfileLanguages'
+// import { ProfileName } from '../../../../components/ProfileName'
+// import { ProfileOccupation } from '../../../../components/ProfileOccupation'
+// import { ProfileWorkPlace } from '../../../../components/ProfileWorkPlace'
 import { RedProfile } from '../../../../types/RedProfile'
-import { ConnectionRequestForm } from './ConnectionRequestForm'
+// import { ConnectionRequestForm } from './ConnectionRequestForm'
 import { getRedProfile } from '../../../../services/auth/auth'
+import { categoriesIdToLabelMap } from '../../../../config/config'
+
+import { Columns, Element, Content, Tag, Heading as BulmaHeading } from 'react-bulma-components'
+import Heading from '../../../../components/atoms/Heading'
+import PipeList from '../../../../components/molecules/PipeList'
+import Read from '../../../../components/molecules/Read'
+import '../../me/Me.scss'
 
 interface Props {
   mentor: RedProfile
-  classes: {
-    avatar: string
-    category: string
-    personalDescription: string
-  }
 }
 
-const styles = (theme: Theme) =>
-  createStyles({
-    avatar: {
-      width: '100px',
-      height: '100px'
-    },
-    category: {
-      color: 'white',
-      fontSize: '12px',
-      margin: '3px',
-      height: '20px'
-    },
-    personalDescription: {
-      marginTop: theme.spacing(6),
-      marginBottom: theme.spacing(6),
-      overflowWrap: 'break-word'
-    }
-  })
-
-export const ProfileMentor = withStyles(styles)(
-  ({ mentor, classes }: Props) => {
+export const ProfileMentor = (
```

Comment:
I think you still have some leftover here ()

Reply:
done